Frontend/ui-customer/src/Screens/NammaSafetyFlow/SafetyEducationScreen/View.purs

```diff
@@ -0,0 +1,269 @@
+cardView cardData index push =
+  linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , padding $ PaddingRight 16
+    , margin $ Margin 16 16 16 0
+    , stroke $ "1," <> Color.grey900
+    , cornerRadius 8.0
+    , gravity CENTER_VERTICAL
+    , onClick push $ const $ ChangeEducationViewIndex index
+    ]
+    [ relativeLayout
+        [ height WRAP_CONTENT
+        , width WRAP_CONTENT
+        , gravity CENTER
+        , margin $ MarginRight 14
+        ]
+        [ imageView
+            [ imageUrl cardData.coverImageUrl
+            , height $ V 90
+            , width $ V 100
+            ]
+        , imageView
+            [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_play_black_white"
+            , height $ V 90
+            , width $ V 100
+            , padding $ Padding 36 31 36 31
+            ]
+        ]
+    , textView
+        $ [ text cardData.title
+          , gravity CENTER_VERTICAL
+          , color Color.black800
+          ]
+        <> FontStyle.body6 TypoGraphy
+    ]
+
+videoView :: forall w. (Action -> Effect Unit) -> NammaSafetyScreenState -> PrestoDOM (Effect Unit) w
+videoView push state =
+  relativeLayout
+    [ width MATCH_PARENT
+    , height MATCH_PARENT
+    , background Color.black900
+    , visibility $ boolToInvisibility $ isJust state.props.educationViewIndex
+    ]
+    [ linearLayout
+        [ height MATCH_PARENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        ]
+        [ linearLayout
+            [ height WRAP_CONTENT
+            , width MATCH_PARENT
+            , background Color.black900
+            , gravity CENTER_VERTICAL
+            ]
+            [ imageView
+                [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_close_white"
+                , height $ V 24
+                , width $ V 24
+                , margin $ Margin 16 16 16 16
+                , onClick push $ const BackPressed
+                ]
+            , textView
+                $ [ text $ getString LEARN_ABOUT_NAMMA_SAFETY
+                  , color Color.white900
+                  , gravity LEFT
+                  , weight 1.0
+                  ]
+                <> FontStyle.h3 TypoGraphy
+            ]
+        , PrestoAnim.animationSet [ triggerOnAnimationEnd true ]
+            $ linearLayout
+                [ width MATCH_PARENT
+                , height MATCH_PARENT
+                , background Color.black900
+                , cornerRadius 8.0
+                , gravity CENTER_VERTICAL
+                , id $ EHC.getNewIDWithTag "SafetyYoutubeVideoView"
+                , onAnimationEnd
+                    ( \action -> do
+                        let
+                          _ = spy "onAnimationEnd" "VideoView"
+                        void $ pure $ runFn5 JB.setYoutubePlayer (EHC.getYoutubeData viewConfig.videoId "PORTRAIT_VIDEO" 1500 true false true) (EHC.getNewIDWithTag "SafetyYoutubeVideoView") "PAUSED" push YoutubeVideoStatus
```

Comment:
Values like `1500 true false true` can be moved to config file of individual screen so that updating the values later will be easier and more clear